Makefile

```diff
@@ -112,7 +112,8 @@ cctest: all
 
 v8:
 	tools/make-v8.sh
-	$(MAKE) -C deps/v8 $(V8_ARCH).$(BUILDTYPE_LOWER) $(V8_BUILD_OPTIONS)
+	$(MAKE) -C deps/v8 $(V8_ARCH).$(BUILDTYPE_LOWER) \
+		$(V8_BUILD_OPTIONS) GYPFLAGS="-Dclang=0"
```

Comment:
Okay, I think this is the issue. By setting `GYPFLAGS` this way, all assignments within the sub-make are ignored.

In the V8 Makefile, [the following line][1] won't append the flag needed:

```make
# i18nsupport=off
ifeq ($(i18nsupport), off)
  GYPFLAGS += -Dv8_enable_i18n_support=0
  TESTFLAGS += --noi18n
endif
```
[1]: https://github.com/nodejs/node/blob/fc6145ff03d98854d372d310e8b80d55bdc72ae2/deps/v8/Makefile#L141

Reply:
Makes sense. So we probably need to create a new build flag in the Makefile.
```make
ifeq ($(clang), off)
  GYPFLAGS += -Dclang=0
endif
```

and use that option when invoking the sub-make.

Reply:
I've attempted to fix this... let us see what happens!